extide/gradle/src/org/netbeans/modules/gradle/api/execute/GradleCommandLine.java

```diff
@@ -60,6 +60,7 @@
  */
 public final class GradleCommandLine implements Serializable {
 
+    private static final String GRADLE_PROJECT_PROPERTY = "gradle-project"; //NOI18N
```

Comment:
Leftover ?
But maybe declare the property as `public` somewhere to have a shared constant.

Reply:
Kind of. Did not want to change the API so late in NB16 cycle, so marked that private and made the two now public non apu one with a comment. Will pick that up for NB17